depends/packages/qt.mk

```diff
@@ -152,6 +151,11 @@ $(package)_config_opts_linux += -fontconfig
 $(package)_config_opts_linux += -no-opengl
 $(package)_config_opts_linux += -no-feature-vulkan
 $(package)_config_opts_linux += -dbus-runtime
+ifneq ($(LTO),)
+# See https://wiki.qt.io/Performance_Tip_Startup_Time
+$(package)_config_opts_linux += -reduce-relocations
```

Comment:
I think we should keep this as `$(package)_config_opts_arm_linux` for now. I'm still seeing build issues for all other Linux HOSTS. i.e for aarch64:
```bash
Trying source 1 (type inline) of library xcb_xinput ...
+ cd /home/ubuntu/bitcoin/depends/work/build/aarch64-linux-gnu/qt/5.15.3-4ca304a51a3/config.tests/xcb_xinput && PKG_CONFIG_SYSROOT_DIR=/ PKG_CONFIG_LIBDIR=/home/ubuntu/bitcoin/depends/aarch64-linux-gnu/lib/pkgconfig /home/ubuntu/bitcoin/depends/work/build/aarch64-linux-gnu/qt/5.15.3-4ca304a51a3/qtbase/bin/qmake "CONFIG -= qt debug_and_release app_bundle lib_bundle" "CONFIG += static warn_off console single_arch" -early "CONFIG += cross_compile" 'QMAKE_USE += xcb_xinput' 'QMAKE_LIBS_XCB_XINPUT = -lxcb-xinput' 'QMAKE_DEPENDS_XCB_XINPUT_CC = XCB' 'QMAKE_DEPENDS_XCB_XINPUT_LD = XCB' 'QMAKE_LIBS_XCB = -L//home/ubuntu/bitcoin/depends/aarch64-linux-gnu/lib -lxcb' 'QMAKE_INCDIR_XCB = //home/ubuntu/bitcoin/depends/aarch64-linux-gnu/include' /home/ubuntu/bitcoin/depends/work/build/aarch64-linux-gnu/qt/5.15.3-4ca304a51a3/config.tests/xcb_xinput
+ cd /home/ubuntu/bitcoin/depends/work/build/aarch64-linux-gnu/qt/5.15.3-4ca304a51a3/config.tests/xcb_xinput && MAKEFLAGS= /usr/bin/gmake
> gmake[1]: Entering directory '/home/ubuntu/bitcoin/depends/work/build/aarch64-linux-gnu/qt/5.15.3-4ca304a51a3/config.tests/xcb_xinput'
> aarch64-linux-gnu-g++ -c -pipe -pipe -std=c++17 -flto -O2 -I/home/ubuntu/bitcoin/depends/aarch64-linux-gnu/include -O2 -w -fPIC  -I. -I/home/ubuntu/bitcoin/depends/aarch64-linux-gnu/include -I/home/ubuntu/bitcoin/depends/work/build/aarch64-linux-gnu/qt/5.15.3-4ca304a51a3/qtbase/mkspecs/linux-aarch64-gnu-g++ -o main.o main.cpp
> aarch64-linux-gnu-g++ -flto -L/home/ubuntu/bitcoin/depends/aarch64-linux-gnu/lib -Wl,-O1 -o xcb_xinput main.o   -lxcb-xinput -L//home/ubuntu/bitcoin/depends/aarch64-linux-gnu/lib -lxcb   
> /usr/lib/gcc-cross/aarch64-linux-gnu/11/../../../../aarch64-linux-gnu/bin/ld: cannot find -lxcb-xinput: No such file or directory
> collect2: error: ld returned 1 exit status
> gmake[1]: *** [Makefile:67: xcb_xinput] Error 1
> gmake[1]: Leaving directory '/home/ubuntu/bitcoin/depends/work/build/aarch64-linux-gnu/qt/5.15.3-4ca304a51a3/config.tests/xcb_xinput'
 => source failed verification.
test config.qtbase_gui.libraries.xcb_xinput FAILED
```

Reply:
Changing of the `-reduce-relocations` flag dropped from this PR for now.

More details available in https://bugreports.qt.io/browse/QTBUG-36129